app/models/feedback_message.rb

```diff
@@ -1,9 +1,9 @@
 class FeedbackMessage < ApplicationRecord
   resourcify
 
-  belongs_to :offender, foreign_key: "offender_id", class_name: "User", optional: true, inverse_of: :offender_feedback_messages
-  belongs_to :reporter, foreign_key: "reporter_id", class_name: "User", optional: true, inverse_of: :reporter_feedback_messages
-  belongs_to :affected, foreign_key: "affected_id", class_name: "User", optional: true, inverse_of: :affected_feedback_messages
+  belongs_to :offender, class_name: "User", optional: true, inverse_of: :offender_feedback_messages
```

Comment:
Since we are on Rails 6, does it just assume these values now based off the object name?

Reply:
It's not related to Rails 6, it was already the default in Rails 5.2. Basically Rails already assumes the default name of the foreign key is `#{name}_id`